src/index.ts

```diff
@@ -7729,7 +7736,7 @@ async function fetchNetworkAccountFromArchiver(): Promise<WrappedAccount> {
       if (!isFromArchiver) {
         throw new Error(`The response signature is not the same from archiver pk:${archiver.publicKey}`)
       }
-      const isResponseVerified = shardus.crypto.verify(res.data)
+      const isResponseVerified = verify(res.data)
```

Comment:
We have the expected public key of the archiver from the archiver list. Please use this as the expected pk and do not trust the pk that is included in the returned object

const isResponseVerified = verify(res.data, archiver.publicKey)